functions/Get-DbaProcess.ps1

```diff
@@ -0,0 +1,168 @@
+﻿Function Get-DbaProcess
+{
+<#
+.SYNOPSIS
+This command displays SQL Server processes.
+
+.DESCRIPTION
+This command displays processes associated with a spid, login, host, program or database.
+
+.PARAMETER SqlServer
+The SQL Server instance.
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. 
+
+.PARAMETER Spids
+This parameter is auto-populated from -SqlServer. You can specify one or more Spids to be displayed.
```

Comment:
Add something like "Also search on BlockingSpid column".
This is related with line 113